src/web/lib/hakuneko/frontend@classic-dark/pages.html

```diff
@@ -532,35 +534,55 @@
              */
             scrollMagic(defaultDistance) {
                 let images = this.$.container.querySelectorAll('.image');
+                // Are we at the end of the page
+                if(images[images.length-1].getBoundingClientRect().bottom-window.innerHeight < 0)
+                {
+                    // Should we go to next chapter because we previouysly reached the end of page ?
+                    if (this.autoNextChapter) { this.requestChapterUp(); return; };
+                    // Prepare for next chapter
+                    this.autoNextChapter = true;
+                    // Todo: Find a way to popup that you have to press spacebar again within 4s
+                    setInterval(function(){ this.autoNextChapter=false; }, 4000);
```

Comment:
May prefer closure instead of function, not quite sure if this is correctly scoped in function context...
`setInterval(() => this.autoNextChapter = false, 4000)`

Reply:
It seems many interval calls could be leaking, each of them ticking every 4 seconds and never cleared, is this intended, or should this be `setTimeout`?

Reply:
Hmmm.... was probably drunk when writing this... setTimeout was indeed intended